src/utils/posthog.ts

```diff
@@ -0,0 +1,11 @@
+import * as fetch from 'node-fetch'
+import { nodePostHog } from 'posthog-js-lite/dist/src/targets/node'
+
+export const posthog = nodePostHog('sTMFPsFhdP1Ssg', {
```

Comment:
should this be coming from a config / env var?

Reply:
It shouldn't - I originally had it as such but Marius had a good point that people deploying the plugin server shouldn't be able to configure this (as it sends metrics to us). 

It also is a public token anyway